portal-api/api-tests/v1/users/post-sign-in-link.api-test.js

```diff
@@ -128,10 +128,45 @@ describe('POST /users/me/sign-in-link', () => {
       expect(sendEmail).not.toHaveBeenCalledWith(EMAIL_TEMPLATE_IDS.SIGN_IN_LINK, userToCreateAsPartiallyLoggedIn.email, expect.anything());
     });
 
-    it('emails a blocked user email', async () => {
+    it('does not email a blocked user email', async () => {
```

Comment:
updated logic around this -- only emails on block rather than every time a blocked user tries